### PR TITLE
#11851 Use new parameter useRealmDefaultLocaleFallback when fetching the realm localization texts

### PR DESCRIPTION
**Pull request by leischt**
Relates to https://github.com/keycloak/keycloak/issues/11851

### CODE REVIEW

**Comment by jonkoops** [comment]
Looks like https://github.com/keycloak/keycloak/pull/11852 will have to be merged first before this will start working.

**Comment by leischt** [comment]
> Looks like [keycloak/keycloak#11852](https://github.com/keycloak/keycloak/pull/11852) will have to be merged first before this will start working.

Yes. I created this PR so that the new Admin UI is not forgotten when this change is done for all other UIs inside of the keycloak/keycloak repository.

File: src/i18n.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ const initOptions = async (
 ): Promise<InitOptions> => {
   const constructLoadPath: LoadPathOption = (_, namespaces) => {
     if (namespaces[0] === "overrides") {
-      return `/admin/realms/${adminClient.realmName}/localization/{{lng}}`;
+      return `/admin/realms/${adminClient.realmName}/localization/{{lng}?useRealmDefaultLocaleFallback=true}`;
```
Comment (jonkoops) [inline src/i18n.ts:20]:
```suggestion
      return `/admin/realms/${adminClient.realmName}/localization/{{lng}}?useRealmDefaultLocaleFallback=true`;
```

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Review by edewit — APPROVED** [review]

**Review by ssilvert — APPROVED** [review]
I merged the other PR.  LGTM.

**Pull request merged**